app/api/routes/recommendation_events.py

```
@@ -0,0 +1,137 @@
+    if event_id:
+        try:
+            # event_id는 문자열이므로 ObjectId로 변환하여 조회
+            event_doc = db_mongo["recommendation_events"].find_one({"_id": ObjectId(event_id)})
+            if event_doc:
+                serialize_object_id(event_doc)
+                event_doc["id"] = event_doc.pop("_id")
+                if hasattr(event_doc.get("timestamp"), "isoformat"):
+                    event_doc["timestamp"] = event_doc["timestamp"].isoformat()
+                
+                return RecommendationEvent(**event_doc)
+        except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)